### DISCUSSION TITLE
im new to this!beginner!

### DISCUSSION THREAD

**Discussion by Learningfrmscratch** [lllyasviel/Fooocus#3588] [Q&A]
can anyone explain me how to use the tool efficiently and properly?
i was trying it from 4,5 days but i was not able to use this properly

**Comment by Crocodilian12** [comment]
You're not giving us enough information to be helpfu.

Kleebz tech has excellent tutorial videos: start there

https://www.youtube.com/watch?v=j1WuQndmgFE&list=PLSofK8pvSk5oHH9iNAAemQREADEY7CPp_

**Comment by Yogesh-DevHub** [comment]
Have you installed the application? There are many YouTube videos explaining fooocus. 

This video series starts from basic prompt using noun, verb and adjectives in fooocus, how to face swap, making character with similar face using lora.
https://youtube.com/playlist?list=PLPFN04WspxqsslRSpiLmwGR8QTpDYNv7z&si=MpvrHsKxcWCuvOKy

**Comment by Learningfrmscratch** [comment]
To install the application I don't have those Nvidia drivers and my laptop
won't support it, which I'm using in colab,runtime is easily getting
disconnected before the output is generated
